examples/todomvc/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+    "serve": "webpack-dev-server"
+  },
+  "devDependencies": {
+    "webpack": "^4.11.1",
+    "webpack-cli": "^3.1.0",
+    "webpack-dev-server": "^3.1.0"
+  }
+}
```

Comment:
Could this copy the `package.json` of other examples to ensure consistency as well>?